piptools/resolver.py

```diff
@@ -629,7 +640,9 @@ def _do_resolve(
         compatible_existing_constraints: dict[str, InstallRequirement],
     ) -> bool:
         """
-        Return true on successful resolution, otherwise remove problematic
+        Actual resolution process.
```

Comment:
This should still start with a verb. Although, this specific phrase seems like a comment rather than a docstring.